Makefile

```diff
@@ -168,19 +168,25 @@ check:
 
 # Run all unit tests. The --files option avoids stashing but passes files; however,
 # the hook setup itself does not pass files to pytest (see .pre-commit-config.yaml).
-.PHONY: test
-test:
-	pre-commit run pytest --hook-stage push --files tests/
+.PHONY: test test-all test-unit test-integration test-performance
+test: test-unit
```

Comment:
Add a comment explaining why we have decided to use `test` as alias for `test-unit`.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/1028/commits/01dfc17976a28b8a25fcbe3006ac452754090483.